snuba/consumers/utils.py

```diff
@@ -0,0 +1,48 @@
+import logging
+import time
+
+from confluent_kafka import KafkaException
+from confluent_kafka.admin import AdminClient
+
+from snuba.utils.streams.configuration_builder import get_default_kafka_configuration
+from snuba.utils.streams.topics import Topic
+
+logger = logging.getLogger("snuba.consumer.utils")
+
+
+class InvalidTopicName(Exception):
+    pass
+
+
+def get_partition_count(topic: Topic) -> int:
+    if not isinstance(topic, Topic):
+        raise InvalidTopicName("Must provide a valid Topic.")
```

Comment:
Isn't this redundant since the type checker would prevent this from being anything other than a Topic?